_posts/2025-01-15-moving-data-between-JBOD-disks-using-cruise-control.md

```diff
@@ -0,0 +1,559 @@
+        {
+          "partitions": [
+            {
+              "partition": "my-topic-2",
+              "size": 0,
+              "offsetLag": 0,
+              "isFuture": false
+            }
+          ],
+          "error": null,
+          "logDir": "/var/lib/kafka/data-2/kafka-log3"
+        },
+        {
+          "partitions": [
+            {
+              "partition": "my-topic-1",
+              "size": 0,
+              "offsetLag": 0,
+              "isFuture": false
+            }
+          ],
+          "error": null,
+          "logDir": "/var/lib/kafka/data-1/kafka-log3"
+        }
+      ]
+    }
+  ],
+  "version": 1
+}
+```
+
+Next, let's move the data from volumes 1 and 2 to volume 0 for all the brokers.
+To do that, we create a `KafkaRebalance` resource and specify `remove-disks` mode.
+```yaml
+apiVersion: kafka.strimzi.io/v1beta2
+kind: KafkaRebalance
+metadata:
+  name: my-rebalance
+  labels:
+    strimzi.io/cluster: my-cluster
+# no goals specified, using the default goals from the Cruise Control configuration
+spec:
+  mode: remove-disks
+  moveReplicasOffVolumes:
+    - brokerId: 3
+      volumeIds:
+        - 1
+        - 2
+    - brokerId: 4
+      volumeIds:
+        - 1
+        - 2
+    - brokerId: 5
+      volumeIds:
+        - 1
+        - 2
+```
+
+Now let’s wait for the `KafkaRebalance` resource to move to `ProposalReady` state. You can check the rebalance summary by running the following command once the proposal is ready:
+```shell
+kubectl get kafkarebalance my-rebalance -n myproject -o yaml
+```
+
+And you should be able to get an output like this:
+```yaml
+apiVersion: kafka.strimzi.io/v1beta2
+kind: KafkaRebalance
+metadata:
+  creationTimestamp: "2025-01-15T10:20:37Z"
+  generation: 1
+  labels:
+    strimzi.io/cluster: my-cluster
+  name: my-rebalance
+  namespace: myproject
+  resourceVersion: "1755"
+  uid: 335ee6b2-59d2-4326-a690-00278d125abd
+spec:
+  mode: remove-disks
+  moveReplicasOffVolumes:
+    - brokerId: 3
+      volumeIds:
+        - 1
+        - 2
+    - brokerId: 4
+      volumeIds:
+        - 1
+        - 2
+    - brokerId: 5
+      volumeIds:
+        - 1
+        - 2
+status:
+  conditions:
+    - lastTransitionTime: "2025-01-15T10:20:52.882813259Z"
+      status: "True"
+      type: ProposalReady
+  observedGeneration: 1
+  optimizationResult:
+    afterBeforeLoadConfigMap: my-rebalance
+    dataToMoveMB: 0
+    excludedBrokersForLeadership: []
+    excludedBrokersForReplicaMove: []
+    excludedTopics: []
+    intraBrokerDataToMoveMB: 0
+    monitoredPartitionsPercentage: 100
+    numIntraBrokerReplicaMovements: 135
+    numLeaderMovements: 0
+    numReplicaMovements: 0
+    onDemandBalancednessScoreAfter: 100
+    onDemandBalancednessScoreBefore: 0
+    provisionRecommendation: ""
+    provisionStatus: UNDECIDED
+    recentWindows: 1
+  sessionId: 028a7dc8-8f6d-485e-8580-93225528b587
+```
+
+Now you can use the `approve` annotation to apply the generated proposal.
```

Comment:
Should we link to the docs for more details about the approval? People wh try it for the first time might find that useful.